app/_hub/kong-inc/oauth2/index.md

```diff
@@ -429,5 +429,17 @@ $ curl -X POST https://your.service.com/oauth2/token \
     --data "refresh_token=XXX"
 ```
 
+----
+
+## gRPC requests
+
+The same access tokens can be used by gRPC applications:
+
+```bash
+$ grpcurl -H 'authorization: bearer XXX' ....
+```
+
+Note that the rest of the credentials flow uses HTTPS and not gRPC protocol.  Depending on your application, you might have to configure the `oauth2` plugin on two separate routes: one under `protocols: ["https"]` and another under `protocols: ["grpcs"]`.
```

Comment:
Could we elaborate more? What types of applications would need this?

Reply:
i honestly don't know.  it depends on how the token is aquired, stored and used.  in short, it's all about the paths used.  since the grpc paths are hardcoded, i assume the provisioning could use a very different one

Reply:
for example, i had to do this in the tests because the provisioning function assumed the endpoint path.